warehouse/warehouse.go

```diff
@@ -1811,6 +1813,15 @@ func Start(ctx context.Context, app app.Interface) error {
 		}))
 	}
 
+	// Report warehouse features
+	go func() {
+		backendconfig.DefaultBackendConfig.WaitForConfig(context.Background())
+		err := features.New(backendconfig.DefaultBackendConfig.Identity()).Send(context.Background(), info.WarehouseFeatures)
```

Comment:
Same comments as the routine in `main.go`. If it makes sense to orchestrate and use the context from above.